### ISSUE TITLE
A11y Code Sprint Berlin: Success Criterion 9.2.4.7 - show carousel controls on keyboard focus

### ISSUE THREAD

**Issue by SarahEdding** [benjaminkott/bootstrap_package#1306]
# Feature Request

## Description

Currently, [carousel](https://bitv-v12.live.typo3.cps.321.works/content-examples/interactive/carousel) controls are focusable but they only show on hover. This needs to change as, even with a visible focus, controls cannot be understood properly if they remain invisible. 

## Solution
When controls or slides receive keyboard focus, make controls visible. 

## Alternative
The carousel has a number of other issues, such as poor colour contrast, a flaw in html semantics, a missing aria-role and no stop/pause button. Recommendation: rebuild carousel according to W3C tutorial https://www.w3.org/WAI/tutorials/carousels/)

**Comment by benjaminkott** [comment]
Controls are now always visible

**Comment by SarahEdding** [comment]
QA done, all fixed
